### PR TITLE
feat: docker-compose write log to file

### PR DESCRIPTION
**Pull request by anjia0532**
Signed-off-by: AnJia <anjia0532@gmail.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] Code compiles correctly.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by 244372610 — DISMISSED** [review]

File: deploy/docker-compose/docker-compose.yaml [hunk 1]
```diff
@@ -53,13 +53,13 @@ services:
     network_mode: host
     restart: always
     healthcheck:
-      test: ["CMD-SHELL", "nc -z 127.0.0.1 65001 || exit 1"]
+      test: ["CMD-SHELL", "/bin/grpc_health_probe -addr=0.0.0.0:65000 || exit 1"]
```
Comment (gaius-qi) [inline deploy/docker-compose/docker-compose.yaml:56]: `/bin/grpc_health_probe -addr=0.0.0.0:65000` => `/bin/grpc_health_probe -addr=:65000`

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: deploy/docker-compose/template/cdn.template.yaml [hunk 2]
```diff
@@ -108,10 +112,10 @@ plugins:
               intervalThreshold: 2h0m0s
 
 # console shows log on console
-console: true
+console: false
 
 # whether to enable debug level logger and enable pprof
-verbose: true
+verbose: false
```
Comment (gaius-qi) [inline deploy/docker-compose/template/cdn.template.yaml:118]: `verbose: true`

**Review by gaius-qi — CHANGES_REQUESTED** [review]

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by jim3ma — APPROVED** [review]
LGTM

**Pull request merged**